typescript/web/src/pages/[workspaceSlug]/datasets/index.tsx

```diff
@@ -24,30 +24,7 @@ import { CookieBanner } from "../../../components/cookie-banner";
 import { WorkspaceTabBar } from "../../../components/layout/tab-bar/workspace-tab-bar";
 import { WorkspaceSwitcher } from "../../../components/workspace-switcher";
 import { NavLogo } from "../../../components/logo/nav-logo";
-
-export const getDatasetsQuery = gql`
-  query getDatasets($where: DatasetWhereInput) {
-    datasets(where: $where) {
-      id
-      name
-      slug
-      images(first: 1) {
-        id
-        url
-        thumbnail500Url
-      }
-      imagesAggregates {
-        totalCount
-      }
-      labelsAggregates {
-        totalCount
-      }
-      labelClassesAggregates {
-        totalCount
-      }
-    }
-  }
-`;
+import { GET_DATASETS_QUERY } from "../../../utils/shared-queries";
 
```

Comment:
This query is too specific to put it in shared queries with such a generic name: it's very specifically tailored for the workspace page listing datasets. with the thumbnail of the first image of a dataset.

I think it should be renamed at least, and then I'm not sure if it should be in shared queries or not.